index.js

```diff
@@ -6,7 +6,12 @@
 var debug = require('debug')('loopback:filter');
 var geo = require('./lib/geo');
 
-module.exports = function filterNodes(nodes, filter) {
+module.exports = {
+  filterNodes: filterNodes,
+  applyFilter: applyFilter,
+};
```

Comment:
This is a breaking change I'd rather avoid.

I am proposing to keep `filterNodes` as the main exported function, and add `applyFilter` as a property of this function.

```js
module.exports = filterNodes;
filterNodes.applyFilter = applyFilter;
```

Thoughts?

Regardless of what approach we decide to take, README needs to be updated to show usage of `applyFilter` too.

Reply:
Sure. Will definitely update the README as well.